app/controllers/initiatives_controller.rb

```diff
@@ -48,16 +56,35 @@ def initiative_params
   end
   
   def find_initiative
-    @initiative = current_user.initiatives.find_by_id(params[:id])
+    team_member
+    @initiative = @team_member.initiatives.find(params[:id])
```

Comment:
If you used `team_member` here instead of `@team_member` could you get rid of line 59?

Reply:
deleted team_member